### PR TITLE
Update Hibernate ORM 5.3.20.Final #1028

### PR DESCRIPTION
**Pull request by btkobayashirun**
Please review #1028.

A modified version of hibernate for #1024 and #1025 .

### CODE REVIEW

File: terasoluna-gfw-parent/pom.xml [hunk 1]
```diff
@@ -591,6 +604,9 @@
     <!-- == MyBatis3 == -->
     <org.mybatis.version>3.5.0</org.mybatis.version>
     <org.mybatis.spring.version>2.0.0</org.mybatis.spring.version>
+    <!-- == Hibernate == -->
+    <!--The version was changed independently as a countermeasure against vulnerabilities of CVE-2020-25638.-->
```
Comment (yoshikawaa) [inline terasoluna-gfw-parent/pom.xml:608]: - [ ] It is good to specify what you changed from.

> Changed from Spring Boot as a countermeasure against CVE-2020-25638.

File: terasoluna-gfw-parent/pom.xml [hunk 2]
```diff
@@ -337,6 +337,19 @@
...
+        <groupId>org.hibernate</groupId>
+        <artifactId>hibernate-core</artifactId>
+        <version>${org.hibernate.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.hibernate</groupId>
+        <artifactId>hibernate-entitymanager</artifactId>
+        <version>${org.hibernate.version}</version>
+      </dependency>
+      <!-- == End Hibernate == -->
```
Comment (yoshikawaa) [inline terasoluna-gfw-parent/pom.xml:351]: By upgrading hibernate-core

- [ ] Transitive dependencies are downgraded by Spring Boot. (Check whether manage version or not.)
* byte-buddy 1.9.11 -> 1.9.7

- [ ] Transitive dependency added. (Need to reflect in each application, guidelines.)
* jaxb-runtime 2.3.1

File: terasoluna-gfw-parent/pom.xml (hunk 1 above)
Comment (btkobayashirun) [reply terasoluna-gfw-parent/pom.xml]: Correspond with 2dd06dd234f7850d9f9840c740a3c2b38e498ec4

File: terasoluna-gfw-parent/pom.xml (hunk 2 above)
Comment (btkobayashirun) [reply terasoluna-gfw-parent/pom.xml:351]: byte-buddy has not been upgraded as it has been confirmed that there are no problems with downgrading.

**Pull request merged**